pkg/rtcp/packet.go

```diff
@@ -0,0 +1,106 @@
+package rtcp
+
+import (
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+)
+
+// RTCP packet types registered with IANA. See: https://www.iana.org/assignments/rtp-parameters/rtp-parameters.xhtml#rtp-parameters-4
+const (
+	TypeSR   = 200 // sender report        RFC 3550, 6.4.1
```

Comment:
If these stay public can you make them the full name `TypeSenderReport` if they stay private it is fine using a less descriptive name (I just want to ensure the GoDoc isn't confusing)

I think they will end up being private if you do an API like.
```
pkt := &rtcp.Nack{SequenceNumber: 5000,  SSRC: 1234}
raw, err := pkt.Marshal()
```

Reply:
Not sure what the API is going to look like yet. I'll make them public and unabbreviated for now.